project/Build.scala

```diff
@@ -465,13 +465,13 @@ object Build {
 
       test in Test := {
         // Exclude legacy tests by default
-        (testOnly in Test).toTask(" -- --exclude-categories=java.lang.Exception").value
+        (testOnly in Test).toTask(" -- --exclude-categories=java.lang.Exception,dotty.tools.dotc.SlowTests").value
```

Comment:
Does this mean that Drone will not run them?

Reply:
Oh, it won't. I will have to find a workaround

Reply:
Done